openslides_backend/action/actions/meeting/import_.py

```diff
@@ -430,3 +519,91 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
             instance["committee_id"],
         ):
             raise MissingPermission(CommitteeManagementLevel.CAN_MANAGE)
+
+    def create_import_create_events(
+        self, instance: Dict[str, Any]
+    ) -> List[CreateEvent]:
+        json_data = instance["meeting"]
+        import_create_events = []
+        for collection in json_data:
+            for entry in json_data[collection].values():
+                # Necessary to remove None-values in event
+                for k, v in list(entry.items()):
+                    if v is None:
+                        entry.pop(k)
```

Comment:
Why? Does the datastore throw an error in that case? Iterating over all entries takes a lot of time so we should only do that if really necessary

Reply:
The BaseEvent in datastore.migartions calls check and the _ModelEvent implements a check, that a field may not have a None value and raises a BadEventException.